README.md

```diff
@@ -7,363 +7,172 @@
-shutter then opens to let the light through and the light hits the film and
-exposes it to make the impression of the scene coming through the lens.
-
-Describing this (simplified) process in a series of steps looks like this:
-
-1. Engage the film mechanism to prepare to roll the film for the next exposure
-2. Roll the film for the exposure to come
-3. Release the film mechanism
-4. Open (flip up) the mirror to redirect the light towards the film (instead of
-   the viewfinder)
-5. Open the shutter (set shutter speed, initialize shutter, activate shutter,
-   release shutter)
-6. Close (flip down) the mirror to direct the incoming light from the lens back
-   to the viewfinder again
-
-Since every manufacturer makes their own hardware, we want to have different
-implementations of the functionality to manage a) the film, b) the mirror and c)
-the shutter. So we create 3 interfaces:
-
-1. `FilmOperations`
-2. `ShutterOperations`
-3. `MirrorOperations`
-
-Putting this together in the `Camera` class gives us the following:
-
-```java
-package com.flatiron.patterns;
-
-public abstract class Camera {
-    private FilmOperations filmOps;
-    private ShutterOperations shutterOps;
-    private MirrorOperations mirrorOps;
-
-    public Camera(FilmOperations filmOps, ShutterOperations shutterOps, MirrorOperations mirrorOps) {
-        this.filmOps = filmOps;
-        this.shutterOps = shutterOps;
-        this.mirrorOps = mirrorOps;
-    }
+clients to specify the types of objects that a generic class should use.
 
-    public void takePhotograph(double shutterSpeed) {
-        Logger.getInstance().log(getName() + " is taking a photograph");
+## Real World Example
 
-        filmOps.engageFilmMechanism();
-        filmOps.rollFilm();
-        filmOps.releaseFilmMechanism();
+Consider the transportation industry. There are plenty of ways to travel and
+get around nowadays! There are cars, buses, planes, trains, and boats just to
+name a few modes of transportation. There are also newer ways to travel - like
+by driving an electric car. Even if we used standard inheritance, the
+implementation of the way we travel may change rather frequently. We might even
+get to a point where implementing a new change is rather difficult in the
+current system. This scenario would greatly benefit from a factory design
+pattern. Let's take this example and see how to implement this pattern!
 
-        mirrorOps.openMirror();;
+## Implementation
 
-        shutterOps.setShutterSpeedSetting(shutterSpeed);
-        shutterOps.initializeShutter();
-        shutterOps.activateShutter();
-        shutterOps.releaseShutter();
+Using the example we described above, we will be implementing the factory
+design pattern. For reference, consider the diagram below:
 
-        mirrorOps.closeMirror();
+![Transportation Factory](https://curriculum-content.s3.amazonaws.com/java-mod-2/factory/Factory-UML.png)
```

Comment:
The image is too compressed. Can you use a higher resolution image?

Reply:
Asking @lizbur10 about this - the original screenshot seems fine. I've tried to reupload it a few different times, but it still seems to be coming out blurry. Standby on the images for this section.